src/main/java/bg/libapp/libraryapp/service/BookService.java

```diff
@@ -119,14 +126,21 @@ private void isBookActive(Book bookToEdit) {
         }
     }
 
+    private void isBookInActive(Book bookToEdit) {
+        if (bookToEdit.isActive()) {
+            logger.error("Book with isbn '" + bookToEdit.getIsbn() + "' is not active!");
```

Comment:
Here the check in the IF is whether the book IS active and the error is that it is not, please, double check this